apps/wallet-dashboard/app/(protected)/staking/page.tsx

```diff
@@ -34,6 +34,8 @@ import { Info } from '@iota/ui-icons';
 import { useMemo } from 'react';
 import { useStakeDialog } from '@/components/Dialogs/Staking/hooks/useStakeDialog';
 import { IOTA_TYPE_ARG } from '@iota/iota-sdk/utils';
+import { useUnstakeDialog } from '@/components/Dialogs/unstake/hooks';
+import { UnstakeDialogView } from '@/components/Dialogs/unstake/enums';
```

Comment:
after adding exports to barrel files we can import all components  from @components on line 19